test/e2e/run_networking_test.go

```diff
@@ -41,6 +43,55 @@ var _ = Describe("Podman run networking", func() {
 
 	})
 
+	It("podman verify network scoped DNS server and also verify updating network dns server", func() {
+		// Following test is only functional with netavark and aardvark
+		SkipIfCNI(podmanTest)
+		SkipIfRemote("not implemented for podman-remote")
```

Comment:
this is implement for remote, no?

Reply:
Yes removed now :) it is implemented for remote